app/components/candidate_interface/application_dashboard_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @application_form.ended_without_success? %>
+  <%= render CandidateInterface::CarryOverBannerComponent.new(application_form: @application_form) %>
+<% end %>
+
+<h1 class="govuk-heading-xl"><%= title %></h1>
+
+<%= render(CandidateInterface::ApplicationDashboardGuidanceComponent.new(application_form: @application_form)) %>
+<% if @application_form.ended_without_success? %>
+  <%= render CandidateInterface::ApplyAgainCallToActionComponent.new(application_form: @application_form) %>
+<% elsif offers_received_and_all_providers_responded? %>
+  <%= render CandidateInterface::OffersCallToActionComponent.new(application_form: @application_form) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
I think these components should also only ever be two thirds of the page width. @frankieroberto 

```suggestion
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    <%= render(CandidateInterface::ApplicationDashboardGuidanceComponent.new(application_form: @application_form)) %>
    <% if @application_form.ended_without_success? %>
      <%= render CandidateInterface::ApplyAgainCallToActionComponent.new(application_form: @application_form) %>
    <% elsif offers_received_and_all_providers_responded? %>
      <%= render CandidateInterface::OffersCallToActionComponent.new(application_form: @application_form) %>
    <% end %>
```

Reply:
Yep, I think these should be in the same two-thirds column as the course choice summary cards below.